src/analyzers/boomanalyzer.h

```diff
@@ -49,9 +44,15 @@ class BoomAnalyzer : public Analyzer::Base {
 
  protected:
   void resizeEvent(QResizeEvent* e);
+  void paletteChange(const QPalette&);
+
+  static const uint COLUMN_WIDTH;
```

Comment:
Can you update the style of all the constants/variables while you're here?
i.e. `COLUMN_WIDTH` -> `kColumnWidth`
`m_bands` -> `bands_`